### ISSUE TITLE
Consultation on training questions

### ISSUE THREAD

**Issue by feiyun1265** [antoine77340/Youtube-8M-WILLOW#9]
1、Can i use video-level features train your model ?
2、Can i use rgb only(no audio) of  frame-level features train your model?
thanks.

**Comment by antoine77340** [comment]
Yes you can train on rgb only but you will need to first specify it in the training command.
Also you will need to modify the architecture so it is not taking into account the audio part (just remove the audio part from the graph). The accuracy should not change a lot without audio.

**Comment by feiyun1265** [comment]
Thanks for reply. @antoine77340. The following commond correct?(use rgb only(no audio) )

1. commond to train video-level feature:
python train.py --train_data_pattern="../dataset/Video_data/video_train/train*.tfrecord" --model=NetVLADModelLF --train_dir=gatednetvladLF-256k-1024-80-0002-300iter-norelu-basic-gatedmoe --frame_features=False --feature_names="mean_rgb" --feature_sizes="1024" --batch_size=80 --base_learning_rate=0.0002 --netvlad_cluster_size=256 --netvlad_hidden_size=1024 --moe_l2=1e-6 --iterations=300 --learning_rate_decay=0.8 --netvlad_relu=False --gating=True --moe_prob_gating=True --max_step=700000

2. commond to train frame-level feature:
python train.py --train_data_pattern="../dataset/Frame_data/frame_train/train*.tfrecord" --model=NetVLADModelLF --train_dir=gatednetvladLF-256k-1024-80-0002-300iter-norelu-basic-gatedmoe --frame_features=True --feature_names="rgb" --feature_sizes="1024" --batch_size=80 --base_learning_rate=0.0002 --netvlad_cluster_size=256 --netvlad_hidden_size=1024 --moe_l2=1e-6 --iterations=300 --learning_rate_decay=0.8 --netvlad_relu=False --gating=True --moe_prob_gating=True --max_step=700000

**Comment by antoine77340** [comment]
Yes, then you need to modify the frame_level_models.py file to take into account the fact that you only have rgb features. It means you will have to do an easy modification of the graph. I am sorry If I cannot help you more I am a bit busy but It should be very easy to modify the architecture of the model to get only RGB.
You can set --max_step=700000 to --max_step=300000, you will get the same performance and it will train faster.

**Comment by feiyun1265** [comment]
I see, thanks for your help.@antoine77340.
